brainscore/metrics/accuracy_delta.py

```diff
@@ -0,0 +1,90 @@
+import itertools
+
+import numpy as np
+
+from brainscore.metrics import Metric, Score
+import random
+from brainscore.metrics.ceiling import SplitHalvesConsistencyBaker
+
+
+class AccuracyDelta(Metric):
+
+    def __call__(self, source, target, image_types, isCeiling=False):
```

Comment:
this violates the [`Metric` interface](https://brain-score.readthedocs.io/en/latest/modules/metrics.html#brainscore.metrics.Metric.__call__): the `__call__` method always accepts two arguments and returns a `Score`.

`image_types` you can probably retrieve from the assembly itself (or you pass in `init`); 
`isCeiling` should be computed in a separate `ceiling` function, e.g. like in https://github.com/brain-score/brain-score/blob/1ecfd7c86f4bda2156b6eee47f272fac1ee1bc1b/brainscore/metrics/image_level_behavior.py#L92